### PR TITLE
Feature/okn 2059

### PR DESCRIPTION
**Pull request by cristain7**
Renamed menu pattern for navbar menu services and refactor it

### CODE REVIEW

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+$osg-hamburger-line-gap: 8px
+
+@function calculate-hamburger-line-position($hamburgerHeight, $lineHeight, $gap)
+  @return (($hamburgerHeight - $lineHeight) / 2) + $gap
+
+.osg-navbar-menu-services
+  nav
+    align-items: baseline
+    display: flex
+    flex-direction: row
```
Comment (simonerlandsen) [inline src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass]: this is the default value, so u do not need to write it.

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+### General
```
Comment (simonerlandsen) [inline src/molecules/navigation/navbar_menu_services/navbar_menu_services.md]: there should be some info on what u (the consumer) need to do urself with javascript, like hiding and showing content

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    .osg-hamburger-button
+      background: none
+      border: none
+      position: relative
+      +osg-mq-tablet-and-up
+        border: 2px solid
+
+      &__text
```
Comment (simonerlandsen) [inline src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass]: u could probably use osg-u-text-4 as a class on the span in the twig to make it look more like in oslo.kommune.no. also the size of the hamburger looks a bit off compared to the oslo.kommune.no one

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass (hunk 1 above)
Comment (cristain7) [reply src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass]: ok

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass (hunk 3 above)
Comment (cristain7) [reply src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass]: I'm extending that  osg-u-text-4 in the css instead

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.md (hunk 2 above)
Comment (cristain7) [reply src/molecules/navigation/navbar_menu_services/navbar_menu_services.md]: I can add a codepen for that

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass (hunk 3 above)
Comment (simonerlandsen) [reply src/molecules/navigation/navbar_menu_services/navbar_menu_services.sass]: good

File: src/molecules/navigation/navbar_menu_services/navbar_menu_services.md (hunk 2 above)
Comment (simonerlandsen) [reply src/molecules/navigation/navbar_menu_services/navbar_menu_services.md]: nice

**Comment by simonerlandsen** [comment]
ok

**Pull request closed without merging**